samples/defer3.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+func f1() {
+	for i := 0; i < 3; i++ {
+		defer fmt.Print(i, " ")
+	}
+}
+
+func f2() {
+	for i := 0; i < 3; i++ {
+		defer func() {
+			fmt.Print(i, " ")
+		}()
+	}
+}
+
+func f3() (result int) {
+	defer func() {
```

Comment:
Same here, a brief description of the behaviour here, try to keep to one line.

Reply:
Done